docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -222,10 +237,10 @@ Attack, Mystic Entangle:
         - Combat
         - Mystic Attacks
     Description: You channel your abilities to tie your target in place.
-    Mechanic: Choose a point within range. All creatures within Aura 3 must make an Agility Save. On a failure, they are Entangled. 
+    Mechanic: Choose a point within range. All creatures within Aura 3 must make an AGL Save. On a failure, they are Entangled. 
```

Comment:
You might check this in a char sheet. I think i made it so that it renders out the Save

Reply:
Updated to reflect what is rendered.